libs/hwauthz/spicedb/spicedb.go

```diff
@@ -217,7 +217,7 @@ func (s *SpiceDBAuthZ) BulkCheck(ctx context.Context, checks []hwauthz.Permissio
 		resc := req.GetResource()
 
 		if pberr := pair.GetError(); pberr != nil {
-			err := fmt.Errorf("spicedb: error while checking permissions: %s", pberr.GetMessage())
+			err := fmt.Errorf("spicedb: error while checking permissions: %s", pberr.GetMessage()) //nolint:err113
```

Comment:
Please explain the use of `nolint` here

Reply:
there were type issues, just had another look at it, I think it was due to a shadowed import of status ("google.golang.org/genproto/googleapis/rpc/status" and , "google.golang.org/grpc/status")